TargetFrameworkDependentValues.props

```diff
@@ -1,15 +1,16 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project>
 
-  <PropertyGroup Condition=" '$(TargetFramework)' != '' And (!$(TargetFramework.StartsWith('nets')) And !$(TargetFramework.StartsWith('net4')) And !$(TargetFramework.StartsWith('monoandroid'))) ">
+  <PropertyGroup Condition=" '$(TargetFramework)' != '' And !$(TargetFramework.StartsWith('nets')) ">
     <JIBuildingForNetCoreApp>True</JIBuildingForNetCoreApp>
   </PropertyGroup>
 
-  <PropertyGroup Condition=" '$(JIBuildingForNetCoreApp)' == 'True' ">
-    <IntermediateOutputPath>$(BaseIntermediateOutputPath)\$(Configuration)-$(TargetFramework.ToLowerInvariant())\</IntermediateOutputPath>
-    <BuildToolOutputFullPath>$(MSBuildThisFileDirectory)bin\Build$(Configuration)-$(TargetFramework.ToLowerInvariant())\</BuildToolOutputFullPath>
-    <ToolOutputFullPath>$(MSBuildThisFileDirectory)bin\$(Configuration)-$(TargetFramework.ToLowerInvariant())\</ToolOutputFullPath>
-    <TestOutputFullPath>$(MSBuildThisFileDirectory)bin\Test$(Configuration)-$(TargetFramework.ToLowerInvariant())\</TestOutputFullPath>
+  <PropertyGroup>
+    <IntermediateOutputPath>$(BaseIntermediateOutputPath)\$(Configuration)-$(DotNetTargetFramework.ToLowerInvariant())\</IntermediateOutputPath>

```

Comment:
Should we "follow convention" and instead use `$(BaseIntermediateOutputPath)\$(Configuration)\$(DotNetTargetFramework.ToLowerInvariant())\`?  Since e.g. `bin/Debug/class-parse.exe` won't be built anymore, the only assemblies in `bin/Debug` will be those for .NET Standard, and there are fewer of those running around…

Reply:
I don't really have an opinion here.  I went back and forth on this and eventually chose to follow the convention we use in `xamarin-android`.  I guess it's nice if you change branches between something that is `net7.0` or `net8.0` they won't clobber each other, but that's not much of a reason.